File: BossFight Rest/BossFight/boss_fight_py_edition/lootItem.py
import math


class LootItem:
    def __init__(self, loot_id: int, loot_name: str, loot_drop_chance: float, cost: int):
        self.loot_id: int = loot_id
        self.loot_name: str = loot_name
        self.loot_drop_chance: float = loot_drop_chance
        self.cost: int = cost

    def set_name(self, name: str):
        self.loot_name = name

    def get_name(self) -> str:
        return self.loot_name

    def get_sell_price(self) -> int:
        return math.ceil(self.cost / 4)

File: BossFight Rest/BossFight/models/Loot/LootItem.cs
using System;

namespace BossFight.Models
{
    public class LootItem
    {
        public int LootId { get; set; }
        public string LootName { get; set; }
        public float LootDropChance { get; set; }
        public int Cost { get; set; }

        public LootItem(int pLootId, string pLootName, float pLootDropChance, int pCost)
        {
            LootId = pLootId;
            LootName = pLootName;
            LootDropChance = pLootDropChance;
            Cost = pCost;
        }

        public void SetName(string pName)
        {
            LootName = pName;
        }

        public string GetName()
        {
            return LootName;
        }

        public int GetSellPrice()
        {
            return Math.Ceiling(Cost / 4);
        }
    }
}